### DISCUSSION TITLE
Timers deleted every time after power off

### DISCUSSION THREAD

**Discussion by Srjosep** [arendst/Tasmota#12751] [Q&A] [answered]
I have an ESP32 wroom flashed with Tasmota. But every time the device is unplugged and plugged again, the timers are gone.
What can be wrong with this? I see there is two modules type to choose in module parameters, ESP32-DevKit (0) and ESP32-DevKit (1).  but I don't know what is the difference

**Comment by sfromis** [comment, marked as answer]
I assume that you mean the timers configured via "Configure Timers". Other timers, like RuleTimer will not be saved. Do check if you've turned off SaveData.
